app/controls/actionlabel.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ACTIONLABEL_H
+#define ACTIONLABEL_H
+
+#include "controls/activelabel.h"
+
+class QAction;
+
+class ActionLabel : public ActiveLabel
```

Comment:
Yes, if it was only used for "HTML source" label in status bar.
Do you have any suggestions for merged class name?